### PR TITLE
Add cas to sign node flow

### PR DESCRIPTION
**Pull request by AvoidMe**

### CODE REVIEW

File: codenotary.repo [hunk 1]
```diff
@@ -0,0 +1,6 @@
+[codenotary]
```
Comment (soksanichenko) [inline codenotary.repo:1]: Does it exist as some remote file which can be downloaded? I guess it will be better than creating repo file by ourself.

Comment (AvoidMe) [reply codenotary.repo]: Good point, fixed

File: sign_node/config.py [hunk 2]
```diff
@@ -64,7 +68,12 @@ def __init__(self, config_file=None, **cmd_args):
...
             "jwt_token": {"type": "string", "nullable": True},
             "dev_pgp_key_password": {"type": "string", "nullable": False},
+            "cas_api_key": {"type": "string", "nullable": True},
+            "cas_signer_id": {"type": "string", "nullable": True},
         }
         super(SignNodeConfig, self).__init__(
             default_config, config_file, schema, **cmd_args
         )
+    
+    def codenotary_enabled(self) -> bool:
```
Comment (anfimovdm) [inline sign_node/config.py:78]: this func can be property

**Review by anfimovdm — APPROVED** [review]

File: requirements.txt [hunk 3]
```diff
@@ -13,3 +13,4 @@ pexpect==4.8.0
 python-gnupg==0.4.8
 websocket-client==0.59.0
 pgpy==0.5.4
+git+https://git.almalinux.org/almalinux/cas_wrapper.git@ALBS-444#egg=cas_wrapper
```
Comment (anfimovdm) [inline requirements.txt:16]: install `cas_wrapper` from git tag `0.0.2`

Comment (AvoidMe) [reply requirements.txt]: Done

File: sign_node/config.py (hunk 2 above)
Comment (AvoidMe) [reply sign_node/config.py:79]: Done

**Review by anfimovdm — APPROVED** [review]

File: sign_node/utils/codenotary.py [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+
+    def __init__(self, api_key, signer_id):
+        self.wrapper = CasWrapper(
+            cas_api_key=api_key,
+            cas_signer_id=signer_id,
+        )
+
+    def verify_artifact(self, package_path: str) -> dict:
+        response = self.wrapper.authenticate_artifact(
+            package_path, return_json=True
```
[the remark this answers is not in the document]
Comment (AvoidMe) [reply sign_node/utils/codenotary.py]: Fixed

**Pull request merged**